### ISSUE TITLE
Variable Support in Raw Body Editor

### ISSUE THREAD

**Issue by kbuzby** [a85/POSTMan-Chrome-Extension#3517]
## App Details:
```
Postman for Chrome
Version 5.2.0
win / x86-64
Chrome 60.0.3112.113
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: seems like it's been there
 2. Expected behaviour: normal variable support in the raw body editor

In a POST request:
- Go to the Body 
- Change to Raw Edit
- Change Type to XML (applicaiton/xml)

When using the raw editor you can use environment and global variables, but you don't get the same assistance as you do elsewhere, where you get autocomplete or even the visibility if the variable is defined globally / in the selected environment.

**Comment by numaanashraf** [comment]
Thanks for the suggestion @kbuzby. We are constantly working on improving the ease of using variables and your feedback helps a lot.

**Comment by gekonwi** [comment]
It would already make a big difference for us if you could just show the regular variable mouse-over / tooltip window (with the initial / current value + scope) for variables in the Raw Body Editor - the same window you show when hovering the mouse over variables in other text fields.

**Comment by gekonwi** [comment]
@vkaegis - any updates on this?

**Comment by vkaegis** [comment]
@gekonwi No ETA on this just yet. Our editor itself requires some tweaking to enable this, so waiting on some prerequisite tasks.

**Comment by Millais** [comment]
Hey @vkaegis is there any progress on this one?

I know our team at PayPal would make big usage of variable autocompletion in the raw request body.

Thanks!

**Comment by Millais** [comment]
Syntax highlighting and variable autocompletion makes for a truly awesome update – thank you @arlemi!

**Comment by arlemi** [comment]
@Millais Glad you like it! Switching our editors to Monaco allows us to do much more with it, and you can expect some more improvements around this in future releases. 🙂

**Comment by JaixTim** [comment]
@arlemi You closed this issue but there is still no tooltip support for showing values of variables in the body as per other fields in the app. Is there a separate issue for that or do I need to raise one?
